api/applications/applicationRouter.js

```diff
@@ -320,183 +170,67 @@ router.post(
- * /application/{approve/:id}:
+ * /application/{update-validate_status/:id}:
  *  put:
- *    summary: Registers profile with okta, updates application ticket "approved" to true, returns a temporary password
- *    description: If a pending application with this profile ID exists, middlware will fetch that profile from the database. This object is passed along into registerOktaUser where its shaped and sent to Oktas API using their clients createUser method.
+ *    summary: Updates validate_status of a given user
+ *    description: user is fetched from MongoDB by the profile_id in req.params, and is modified by the validate_status selected by the user (approved/rejected)
  *    tags:
  *      - application
  *    security:
- *      - okta: [authRequired, adminRequired]
+ *      - auth0: [authRequired, adminRequired]
  *    parameters:
  *      - in: param
  *        name: profile id
- *        schema:
- *          type: string
- *        description: Profile ID of pending applicant
  *    responses:
  *      '200':
- *        description: Response from successful put
+ *        description: Response from successful post to MongoDB
  *        content:
  *          application/json:
  *            schema:
  *              type: object
- *              properties:
- *                tempPassword:
- *                  description: temporary password for admins to give to approved users
- *                  type: string
  *              items:
- *                $ref: '#/components/schemas/Application'
- *              example:
- *                message: "3sdfi324"
+ *                status: 200
+ *                message: {
+ *                  result: true
+ *                    }
  *      '401':
  *        $ref: '#/components/responses/UnauthorizedError'
- */
 
-// registers a user with okta and approves their application ticket
+NOTE: FE discerns if the user is a mentor/mentee and sends back the appropriate shape: 
+mentor = {validate_status & current_company }
+mentee = {validate_status}
+this endpoint can then send the validate status to the appropriate endpoint depending on whether the current_company is present or not
+  
+authRequired, and AdminRequired imports have been left (commented out) because they will likely be needed when auth0 is implemented.
 
-router.put(
-  '/approve/:id',
-  authRequired,
-  adminRequired,
-  checkApplicationExists,
-  findProfile,
-  registerOktaUser,
-  (req, res, next) => {
-    const application_id = req.application.application_id;
+*/
+router.post('/update-validate_status/:profile_id', (req, res, next) => {
+  const isMentor = req.body.current_company;
+  const { profile_id } = req.params;
 
-    Application.updateTicket(application_id, { approved: true })
-      .then(() => {
-        res.status(200).json({
-          tempPassword: req.tempPassword,
-        });
+  if (isMentor) {
+    axios
+      .post(`${baseURL}/update/mentor/${profile_id}`, {
+        validate_status: req.body.validate_status,
```

Comment:
A middleware should also be used to check if the body has all the fields that you're expecting so that you can provide an appropriate error response (422 - body not complete) and avoid making a call to the DS DB if it would only result in an error.